Tests/LibC/TestLibCMkTemp.cpp

```diff
@@ -121,3 +121,57 @@ TEST_CASE(test_mkstemp_unique_filename)
 
     munmap(ptr, sizeof(*ptr));
 }
+
+TEST_CASE(test_mkstemps_unique_filename)
+{
+    u8* ptr = (u8*)mmap(nullptr, 0x1000, PROT_READ | PROT_WRITE, MAP_SHARED | MAP_ANONYMOUS, -1, 0);
+    EXPECT_NE(ptr, MAP_FAILED);
+
+    if (fork() == 0) {
+        char path[] = "/tmp/test.mkstemps.__prefix__XXXXXX__suffix__";
```

Comment:
Andreas just made a comment live

> I dont understand why you need to fork in a test

Its a question, I believe he will still merge it, unless there is a follow to that comment

Reply:
The test was just copied from the `mkstemp` test. It verifies that two processes asking for the same path will get a different random path. It is not necessary to do this for both `mkstemps` and `mkstemp`, so I'll simplify the test.

Reply:
After looking at the test again I noticed all tests in `TestLibCMkTemp.cpp` do a fork, to check the generated paths for uniqueness, so I think it's fine like this. Unless this is not by design but the tests ended up looking the same just from copy-pasting..